google-cloud-spanner/src/main/java/com/google/cloud/spanner/v1/stub/metrics/MetricRegistryConstants.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spanner.v1.stub.metrics;
```

Comment:
Sorry, I didn't notice this before now: Is there a specific reason that this file is placed in this package? The `com.google.cloud.spanner.v1` package and sub-packages only contain generated source files and shouldn't contain any other files. (I'm not 100% sure, but there is a good chance that re-generating the gapic client will also automatically delete all files in the package, which would mean that this file would also be deleted.)

Reply:
done in 8548f8a3cff8fba7778805bdc6161454bbfdc305